packages/react-components/react-spinbutton/src/stories/SpinButton/index.stories.tsx

```diff
@@ -21,7 +20,7 @@ const meta: Meta = {
   parameters: {
     docs: {
       description: {
-        component: [descriptionMd, bestPracticesMd, accessibilityMd].join('\n'),
```

Comment:
How about linking from the SpinButton to the a11y docs

Reply:
Yup, it’s in the plan! I’m going to be adding an a11y authoring section next to best practices that will link to the full descriptive spec